.github/workflows/scripts/disable_statements_check.py

```
@@ -25,6 +25,14 @@
 class DisableStatementsChecker:
     """Checker for various disable statements in code files."""
 
+    BINARY_EXTENSIONS = {
+        ".png",
+        ".jpg",
+        ".jpeg",
+        ".gif",
+        ".webp",
+    }
```

Ruff (0.14.14)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)